<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cheker</class>
 <widget class="QMainWindow" name="Cheker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cheker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="ValidEmailsTable">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>281</width>
      <height>431</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>login</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>password</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="ShowValidLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Valid e-mails</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QLabel" name="DomainLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Supported domains:</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>28</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="DelimiterLabel">
      <property name="text">
       <string>Delimiter:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="DilimiterSetLineEdit">
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string>:</string>
      </property>
      <property name="maxLength">
       <number>1</number>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="ForcedModeCHB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Forced mode(skip errors)</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>89</y>
       <width>151</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QCheckBox" name="YandexCHB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>@yandex.ru</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="MailRuCHB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>@mail.ru</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="RamblerCHB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>@rambler.ru</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="BKCHB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>@bk.ru</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="OtherCHB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>unknown</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QLabel" name="yandexCountLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLabel" name="mailCountLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLabel" name="ramblerCountLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLabel" name="bkCountLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLabel" name="otherCountLabel">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Good</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLCDNumber" name="GoodLCD">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Bad</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Total</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLCDNumber" name="TotalLCD">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLCDNumber" name="BadLCD">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>splitter_3</zorder>
    <zorder>DomainLabel</zorder>
    <zorder>ForcedModeCHB</zorder>
   </widget>
   <widget class="QLabel" name="MainProcessLable">
    <property name="geometry">
     <rect>
      <x>206</x>
      <y>473</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Main Process</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="ProgressBar">
    <property name="geometry">
     <rect>
      <x>203</x>
      <y>500</y>
      <width>650</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="CurrentProcess">
    <property name="geometry">
     <rect>
      <x>207</x>
      <y>503</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Process</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="1">
      <widget class="QPushButton" name="CheckButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Check</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="CloseBaseButton">
       <property name="text">
        <string>Close Base</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="OpenBaseButton">
       <property name="text">
        <string>Open Base</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="testButton">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="SearchGB">
    <property name="geometry">
     <rect>
      <x>499</x>
      <y>29</y>
      <width>341</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Search</string>
    </property>
    <widget class="QComboBox" name="ListToSearchCB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="TextSearchLE">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="AddButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DelButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SerchButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="TabSearchRes">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>140</y>
      <width>341</width>
      <height>321</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Base"/>
    <addaction name="actionClose_Base"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Base">
   <property name="text">
    <string>Open Base</string>
   </property>
  </action>
  <action name="actionClose_Base">
   <property name="text">
    <string>Close Base</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
